poppy/tests/test_wfe.py

```diff
@@ -86,3 +89,68 @@ def test_ParameterizedAberration():
 
     assert stddev < 1e-16, ("ParameterizedAberration disagrees with "
                             "ZernikeAberration! stddev {}".format(stddev))
+
+def test_KolmogorovWFE_stats():
+    # verify statistics of random numbers
+    KolmogorovWFE = wfe.KolmogorovWFE(Cn2=CN2, dz=DZ)
+    npix = 2048
+    a = KolmogorovWFE.rand_turbulent(npix)
+    b = KolmogorovWFE.rand_symmetrized(npix, 1)
+    c = KolmogorovWFE.rand_symmetrized(npix, -1)
+    
+    assert(np.round(np.abs(np.mean(a)), 2) == np.round(0.0, 2))
+    assert(np.round(np.var(a), 2) == np.round(1.0, 2))
+    assert(np.round(np.mean(b), 2) == np.round(0.0, 2))
+    assert(np.round(np.var(b), 2) == np.round(1.0, 2))
+    assert(np.round(np.mean(c), 2) == np.round(0.0, 2))
+    assert(np.round(np.var(c), 2) == np.round(1.0, 2))
+
+def test_KolmogorovWFE_Cn2():
+    # verifiy correct calculation of Cn2 from Fried parameter
```

Comment:
very minor: wrong spelling of 'verify' in comment, here and in the following function